tools/lint-fix.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Copyright (c) 2018-2020 MobileCoin Inc.
+
+set -e
+
+if [[ ! -z "$1" ]]; then
+    cd "$1"
+fi
+
+for toml in $(grep --exclude-dir cargo --exclude-dir rust-mbedtls --exclude-dir mobilecoin --exclude-dir ledger-mob --include=Cargo.toml -r . -e '\[workspace\]' | cut -d: -f1); do
```

Comment:
Optional/Equivalent: Replace `cut` with `grep` flag `-l`
```suggestion
for toml in $(grep --exclude-dir cargo --exclude-dir rust-mbedtls --exclude-dir mobilecoin --exclude-dir ledger-mob --include=Cargo.toml -r . -e '\[workspace\]' -l); do
```